sass/base/_margins.scss

```diff
@@ -0,0 +1,7 @@
+.margin-100 {
+    margin: .6rem 0 1.4rem 0;
+}
+
+.margin-200 {
+    margin: 2.6rem 0 1.3rem 0;
+}
```

Comment:
jeśli to są marginesy które używa się tylko raz w projekcie, to można je przenieść bezpośrednio do danej sekcji.

Reply:
w momencie kiedy już każdy napisał swoje marginesy i paddingi nie są potrzebne jako osobne klasy